src/lazy/expanded/template.rs

```diff
@@ -434,14 +435,10 @@ impl<'top, D: Decoder> Iterator for TemplateSequenceIterator<'top, D> {
                 TemplateBodyExprKind::MacroInvocation(body_invocation) => {
                     // ...it's a TDL macro invocation. Resolve the invocation to get a reference to the
                     // macro being invoked.
-                    let invoked_macro = self
-                        .context
-                        .macro_table()
-                        .macro_at_address(body_invocation.invoked_macro_address)
-                        .unwrap();
+                    let invoked_macro = body_invocation.invoked_macro.as_ref();
```

Comment:
🗺️ There were several places where a macro address would need to be resolved into a macro reference. These are no longer needed because the macro definitions hold an `Rc`.